src/components/include/ToggleSwitch.js

```diff
@@ -1,18 +1,54 @@
-import $ from 'jquery';
-import React from 'react';
-import setClass from 'classnames';
+import setClass from "classnames";
+import $ from "jquery";
+import PropTypes from "prop-types";
+import React from "react";
 
-function ToggleSwitch(props) {
+const ToggleSwitch = function ToggleSwitch({
+    cls,
+    color,
+    id,
+    title,
+    name,
+    checked,
+    disabled,
+    onStateChange,
+}) {
+    const { htmlFor } = "form-" + id + "-label";
```

Comment:
We probably don't need braces around `htmlFor` here

Reply:
Good catch